.travis.yml

```diff
@@ -17,21 +17,24 @@ services:
 matrix:
   fast_finish: true
 
-env:
-  - TEST_RUN="./tests/test-docker-compose.sh"
-  - TEST_RUN="./tests/test-minikube.sh"
-  - TEST_RUN="./tests/test-kubernetes.sh"
-
 before_install:
   - sudo apt-get install shellcheck
-  - pip install -U -r test-requirements.txt
-
-install:
-  - "./scripts/install.sh"
-  - "./scripts/bx_auth.sh"
+  - sudo pip install yamllint
+  - git clone https://github.com/IBM/pattern-ci
 
 before_script:
-  - "./tests/test-shellcheck.sh"
-  - "./tests/test-yamllint.sh"
-
-script: "$TEST_RUN"
+  - "./pattern-ci/tests/shellcheck-lint.sh"
+  - "./pattern-ci/tests/yaml-lint.sh"
+
+jobs:
+  include:
+    - install:
+        - ./pattern-ci/scripts/install-ibmcloud.sh
+        - ./pattern-ci/scripts/bx-auth.sh
```

Comment:
You can also remove the `install-ibmcloud.sh` and `bx-auth.sh` scripts since this stage is for a local docker-compose